### PR TITLE
Add monitor retry for UDP servers and clean timeouts and intervals when ...

### PR DESCRIPTION
**Pull request by willyaranda**
...we are closing

### CODE REVIEW

**Comment by willyaranda** [comment]
@frsela review?

File: src/ns_msg_mon/msg_mon_server.js [hunk 1]
```diff
@@ -54,13 +54,42 @@ monitor.prototype = {
     });
 
     // Check if we are alive
-    setTimeout(function() {
-      if (!self.ready)
+    this.readyTimeout = setTimeout(function() {
+      if (!self.ready) {
         log.critical(log.messages.CRITICAL_NOTREADY);
+      }
     }, 30 * 1000); //Wait 30 seconds
```
Comment (frsela) [inline src/ns_msg_mon/msg_mon_server.js:61]: Why the time is hardcoded instead configurable?

Comment (willyaranda) [inline src/ns_msg_mon/msg_mon_server.js:61]: This is the time where we think that should wait to say: "hey, I am ready". I can put a pref, but I don't think it is worth it.

Comment (frsela) [inline src/ns_msg_mon/msg_mon_server.js:61]: Damn

**Pull request merged**